Manifests/ManagedPreferencesApplications/com.vpntracker.365mac-config.plist

```diff
@@ -0,0 +1,344 @@
+			<key>pfm_name</key>
+			<string>Managed_Onboarding_TeamAPIToken</string>
+			<key>pfm_require</key>
+			<string>always</string>
+			<key>pfm_title</key>
+			<string>Team API Token</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>Greeting text to show instead of the default "Welcome, {name}".</string>
+			<key>pfm_name</key>
+			<string>Managed_Onboarding_Greeting</string>
+			<key>pfm_title</key>
+			<string>Greeting text</string>
+			<key>pfm_type</key>
+			<string>string</string>
+			<key>pfm_value_placeholder</key>
+			<string>Welcome!</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>A comma-separated list of email domains that users are allowed to use when joining your team.</string>
+			<key>pfm_name</key>
+			<string>Managed_Onboarding_EmailDomains</string>
+			<key>pfm_title</key>
+			<string>Allowed email domains</string>
+			<key>pfm_type</key>
+			<string>string</string>
+			<key>pfm_value_placeholder</key>
+			<string>yourcompany.example, subdomain.example.com, *.yourcompany.example</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>You can optionally set the email address the user will use to sign in to VPN Tracker, when deploying a personalized profile.</string>
+			<key>pfm_name</key>
+			<string>Managed_Onboarding_Email</string>
+			<key>pfm_title</key>
+			<string>Email address</string>
+			<key>pfm_type</key>
+			<string>string</string>
+			<key>pfm_value_placeholder</key>
+			<string>user@example.com</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>Used in the onboarding greeting and during VPN Tracker account registration for new users.</string>
+			<key>pfm_name</key>
+			<string>Managed_Onboarding_GivenName</string>
+			<key>pfm_title</key>
+			<string>Given name</string>
+			<key>pfm_type</key>
+			<string>string</string>
+			<key>pfm_value_placeholder</key>
+			<string>Grace</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>Used in the onboarding greeting and during VPN Tracker account registration for new users.</string>
+			<key>pfm_name</key>
+			<string>Managed_Onboarding_FamilyName</string>
+			<key>pfm_title</key>
+			<string>Surname</string>
+			<key>pfm_type</key>
+			<string>string</string>
+			<key>pfm_value_placeholder</key>
+			<string>Hopper</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>Full name of the user displayed during onboarding. If given name and family name are set, this value is ignored.</string>
+			<key>pfm_name</key>
+			<string>Managed_Onboarding_FullName</string>
+			<key>pfm_title</key>
+			<string>Full name</string>
+			<key>pfm_type</key>
+			<string>string</string>
+			<key>pfm_value_placeholder</key>
+			<string>Grace Hopper</string>
+		</dict>
+	</array>
+	<key>pfm_targets</key>
+	<array>
+		<string>user</string>
+	</array>
```

Comment:
Are you certain that this is only in the user-level profile context?

Reply:
No, I'm not. I tried reading up what this value actually means but failed to find a good explanation. As far as I understand, `system` is also called device-level and means that a profile can be installed system-wide and applies to all users, while `user` just targets a specific user and does not affect other users on a system. Is that correct? If so, both levels would probably be applicable to our app.

Reply:
That is correct.  FWIW, I'm not entirely sure in this day and age why someone would configure a profile to be user-specific, because at the end of the day if it's your org's device you probably want anyone within your org to use it.

So you can go ahead and add the `<string>system</string>` option here as well.